packages/gatsby/src/query/index.js

```diff
@@ -97,6 +97,9 @@ const popNodeQueries = state => {
 
     return dirtyIds
   }, new Set())
+
+  // boundActionCreators.deleteComponentsDependencies([...uniqDirties])
```

Comment:
This is commented out fix - next commit will uncomment it.

Reply:
next commit that isn't linting fix that is 😅

Reply:
Check https://circleci.com/gh/gatsbyjs/gatsby/418102?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link for example of failed test before fix was applied